packages/lit-analyzer/src/analyze/util/type/is-assignable-in-attribute-binding.ts

```diff
@@ -31,6 +32,21 @@ export function isAssignableInAttributeBinding(
 			];
 		}
 	} else {
+		if (assignment.kind !== HtmlNodeAttrAssignmentKind.STRING) {
+			// Purely static attributes are never security checked, they're handled
+			// in the lit-html internals as trusted by default, because they can
+			// not contain untrusted data, they were written by the developer.
+			//
+			// For everything else, we may need to apply a different type comparison
+			// for some security-sensitive built in attributes and properties (like
+			// <script src>).
+			const securityDiagnostics = isAssignableBindingUnderSecuritySystem(htmlAttr, { typeA, typeB }, request, "no-incompatible-type-binding");
+			if (securityDiagnostics !== undefined) {
+				// The security diagnostics are binding. Note that this may be an
```

Comment:
Should this be "for the binding", or am I missing another meaning of "are binding"?

Reply:
Good point, using the word "binding" here is confusing because of the more salient other meaning. Clarified